src/system/AdminModule/Controller/AdminInterfaceController.php

```diff
@@ -0,0 +1,785 @@
+			$cid = empty($requested_cid) ? $this->getVar('startcategory') : $requested_cid;			
+		}else{			
+			$cid = \ModUtil::apiFunc('ZikulaAdminModule', 'admin', 'getmodcategory', array('mid' => $caller['info']['id']));
+		}			
+		$caller['category'] = \ModUtil::apiFunc('ZikulaAdminModule', 'admin', 'getCategory', array('cid' => $cid));
+		
+		return ['caller' => $caller];
+	}
+	
+	/**
+	 * @Route("/developernotices")
+	 *
+	 * @Template()
+	 * @Theme("admin")
+	 *
+	 * Open the admin container
+	 *
+	 * @return Response symfony response object
+	 */
+	public function developernoticesAction()
+	{
+		if (!$this->hasPermission('ZikulaAdminModule::', '::', ACCESS_ADMIN)) {
+			throw new AccessDeniedException();
+		}		 
+
+		$modvars = \ModUtil::getVar('ZikulaThemeModule');
+		
+		$data = array();
+		$data['devmode'] = $this->get('kernel')->getEnvironment() === 'dev';
+		
+		if ($data['devmode'] == true) {
+				$data['cssjscombine'] = $modvars['cssjscombine'];
+		
+				if ($modvars['render_compile_check']) {
+					$data['render']['compile_check'] = array('state' => $modvars['render_compile_check'],
+							'title' => $this->__('Compile check'));
+				}
+				if ($modvars['render_force_compile']) {
+					$data['render']['force_compile'] = array('state' => $modvars['render_force_compile'],
+							'title' => $this->__('Force compile'));
+				}
+				if ($modvars['render_cache']) {
+					$data['render']['cache']         = array('state' => $modvars['render_cache'],
+							'title' => $this->__('Caching'));
+				}
+				if ($modvars['compile_check']) {
+					$data['theme']['compile_check']  = array('state' => $modvars['compile_check'],
+							'title' => $this->__('Compile check'));
+				}
+				if ($modvars['force_compile']) {
+					$data['theme']['force_compile']  = array('state' => $modvars['force_compile'],
+							'title' => $this->__('Force compile'));
+				}
+				if ($modvars['enablecache']) {
+					$data['theme']['cache']          = array('state' => $modvars['enablecache'],
+							'title' => $this->__('Caching'));
+				}
+		}
+
+		return ['developer' => $data];
+	}
+	
+	/**
+	 * @Route("/securityanalyzer")
+	 *
+	 * @Template()
+	 * @Theme("admin")
+	 *
+	 * Open the admin container
+	 *
+	 * @return Response symfony response object
+	 */
+	public function securityanalyzerAction()
+	{
+		$data = array();
+	
+		// check for magic_quotes
+		$data['magic_quotes_gpc'] = \DataUtil::getBooleanIniValue('magic_quotes_gpc');
+	
+		// check for register_globals
+		$data['register_globals'] = \DataUtil::getBooleanIniValue('register_globals');
+	
+		// check for config.php being writable
+		$data['config_php'] = (bool)is_writable('config/config.php');
+	
+		// check for .htaccess in temp directory
+		$app_htaccess = false;
+		$appDir = $this->get('kernel')->getRootDir();
+		if ($appDir) {
+			// check if we have an absolute path which is possibly not within the document root
+			$docRoot = \System::serverGetVar('DOCUMENT_ROOT');
```

Comment:
This is probably available from the `Request`? or some symfony config? (again `System` is deprecated)